vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -487,7 +455,7 @@ for parameters](../user-guides/parameters.md).
                         options=["#ff5267", "#3949ab"], multi=False, value="#3949ab", title="Color Virginica"),
                     ),
                 vm.Parameter(
-                    targets=["scatter_iris.opacity", "hist_iris.opacity"],
+                    targets=["scatter_iris.opacity"],
```

Comment:
This is correct how you've changed it here! I think you have to also remove it from the code section though! :) I think the two is currently out of sync between the tab **code** and **app.py**
<img width="680" alt="Screenshot 2024-02-28 at 13 40 13" src="https://github.com/mckinsey/vizro/assets/90609403/7da1561e-3c55-4bff-8e1d-2d117d6bf1ef">